test/functional/test_framework/util.py

```diff
@@ -56,6 +56,11 @@ def assert_greater_than(thing1, thing2):
         raise AssertionError("%s <= %s" % (str(thing1), str(thing2)))
 
 
+def assert_no_key(key, dictionary):
```

Comment:
Is this necessary? Can't just write `assert(key not in dict)` inline?

Reply:
It is not. I'll change it. Thanks

Reply:
It is not. Good idea. Thanks.